recipes-wpe/wpeframework/wpeframework-clientlibraries_git.bb

```diff
@@ -0,0 +1,75 @@
+SUMMARY = "WPEFramework client libraries"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=f1dffbfd5c2eb52e0302eb6296cc3711"
+PR = "r0"
+
+require include/wpeframework.inc
+
+SRC_URI = "git://github.com/rdkcentral/ThunderClientLibraries.git;protocol=git;branch=master"
+SRCREV = "bea2273b40525b01cc306e4acb03d13bf3d79313"
+
+# ----------------------------------------------------------------------------
+
+include include/compositor.inc
+
+DEPENDS = " \
+    wpeframework wpeframework-interfaces \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', '${WPE_COMPOSITOR_DEP}', '', d)} \
+"
+
+WPE_CDMI_ADAPTER_IMPL ?= "${@bb.utils.contains('DISTRO_FEATURES', 'nexus_svp', 'opencdmi_brcm_svp', 'opencdm_gst', d)}"
+
+PACKAGECONFIG ?= " \
+    release \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'opencdm', 'opencdm ${WPE_CDMI_ADAPTER_IMPL}', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'provisioning', 'provisionproxy', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'security', 'securityagent', '', d)} \
+    virtualinput \
+    "
+
+# Buildtype
+PACKAGECONFIG[debug]          = "-DBUILD_TYPE=Debug,,"
+PACKAGECONFIG[debugoptimized] = "-DBUILD_TYPE=DebugOptimized,,"
+PACKAGECONFIG[releasesymbols] = "-DBUILD_TYPE=ReleaseSymbols,,"
+PACKAGECONFIG[release]        = "-DBUILD_TYPE=Release,,"
+PACKAGECONFIG[production]     = "-DBUILD_TYPE=Production,,"
+
```

Comment:
Buildtype is only required in wpeframework code.

Reply:
Removed Buildtype.